### PR TITLE
feat: As a user, I want to be able to see the schema details page on Explorer

### PR DESCRIPTION
**Pull request by voenkomatiwe**
## What does this PR do?

#416

### CODE REVIEW

File: explorer/src/components/NotFoundPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <span>{pageData.title}</span>
+          {pageData.showId && (
+            <span className="max-w-[100px] md:max-w-[200px] text-[#3d3d50] overflow-hidden text-ellipsis">
+              {decodedId}
+            </span>
+          )}
+          <span>hasn`t been found</span>
+        </p>
+        <Link
+          to={APP_ROUTES.ATTESTATIONS}
```
Comment (voenkomatiwe) [inline explorer/src/components/NotFoundPage/index.tsx:28]: direction must be passed to the component

**Review by ars9 — APPROVED** [review]
Looks good.

File: explorer/src/pages/Schema/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    revalidateOnFocus: false,
+  });
+
+  //todo add loading
+  if (isLoading || isValidating) return <p>Loading...</p>;
+  if (!schema) return <NotFoundPage page="schema" id={id} />;
+  return (
+    <section className="flex flex-col gap-6 w-full mb-10 md:mb-20 xl:max-w-[1200px] xl:m-auto">
+      <div className="flex flex-col px-5 md:px-10 gap-6">
+        <Back to={APP_ROUTES.SCHEMAS} />
```
Comment (Solniechniy) [inline explorer/src/pages/Schema/index.tsx:33]:
```suggestion
        <Back to={() => navigate(-1)} />
```

File: explorer/src/pages/Schema/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import { SWRKeys } from "@/interfaces/swr/enum";
+import { useNetworkContext } from "@/providers/network-provider/context";
+import { APP_ROUTES } from "@/routes/constants";
+
+import { RecentAttestations } from "./RecentAttestations";
+
+export const Schema = () => {
+  const { id } = useParams();
+  const { sdk } = useNetworkContext();
+
```
Comment (Solniechniy) [inline explorer/src/pages/Schema/index.tsx:17]: const navigate = useNavigate()

**Review by Solniechniy — CHANGES_REQUESTED** [review]

**Review by Solniechniy — APPROVED** [review]

**Pull request merged**